package.json

```diff
@@ -7,8 +7,8 @@
   "react-native": "dist-native/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc --emitDeclarationOnly && webpack src/index.ts --mode production --resolve-extensions .ts --module-bind ts=ts-loader --define process.env.BROWSER=1 --output-filename index.js --output-library-target umd",
-    "build-for-script-tag": "tsc --emitDeclarationOnly && webpack src/browser.js --mode production --resolve-extensions .ts --module-bind ts=ts-loader --define process.env.BROWSER=1 --woutput-filename browser.js --output-library-target umd",
+    "build": "tsc --emitDeclarationOnly && webpack --config webpack.browserConfig.js",
+    "build-for-script-tag": "tsc --emitDeclarationOnly && webpack --config webpack.scriptConfig.js",
```

Comment:
All this config now lives in the config files, This was required since webpack-cli 4